docs/UserGuide.md

```diff
@@ -954,7 +955,6 @@ Exit | `exit`
 
 Term | Definition
 -----|-----------
-ASCII | Characters that are recognised by a computer.
```

Comment:
No more CELC definitions?

Reply:
I decided it might be best to remove it as the above definition is inconsistent with the definition earlier in the user guide where the full list of accepted ASCII characters was shown.